### ISSUE TITLE
Stored Object: paper:arxiv.2310.18313

### ISSUE THREAD

**Issue by zzmtsvv** [zzmtsvv/papers-feed#4]
{
  "sourceId": "arxiv",
  "paperId": "2310.18313",
  "url": "https://arxiv.org/abs/2310.18313",
  "title": "FP8-LM: Training FP8 Large Language Models",
  "authors": "Houwen Peng, Kan Wu, Yixuan Wei, Guoshuai Zhao, Yuxiang Yang, Ze Liu, Yifan Xiong, Ziyue Yang, Bolin Ni, Jingcheng Hu, Ruihang Li, Miaosen Zhang, Chen Li, Jia Ning, Ruizhe Wang, Zheng Zhang, Shuguang Liu, Joe Chau, Han Hu, Peng Cheng",
  "abstract": "In this paper, we explore FP8 low-bit data formats for efficient training of large language models (LLMs). Our key insight is that most variables, such as gradients and optimizer states, in LLM training can employ low-precision data formats without compromising model accuracy and requiring no changes to hyper-parameters. Specifically, we propose a new FP8 automatic mixed-precision framework for training LLMs. This framework offers three levels of FP8 utilization to streamline mixed-precision and distributed parallel training for LLMs. It gradually incorporates 8-bit gradients, optimizer states, and distributed learning in an incremental manner. Experiment results show that, during the training of GPT-175B model on H100 GPU platform, our FP8 mixed-precision training framework not only achieved a remarkable 39% reduction in real memory usage but also ran 75% faster than the widely adopted BF16 framework (i.e., Megatron-LM), surpassing the speed of Nvidia Transformer Engine by 37%. This largely reduces the training costs for large foundation models. Furthermore, our FP8 mixed-precision training methodology is generic. It can be seamlessly applied to other tasks such as LLM instruction tuning and reinforcement learning with human feedback, offering savings in fine-tuning expenses. Our FP8 low-precision training framework is open-sourced at {this https URL}{this http URL}.",
  "timestamp": "2025-08-08T18:18:03.690Z",
  "rating": "novote",
  "publishedDate": "2023/10/27",
  "tags": [
    "Machine Learning (cs.LG)",
    "Computation and Language (cs.CL)"
  ],
  "doi": "",
  "journalName": "",
  "sourceType": "url"
}

**Comment by zzmtsvv** [comment]
{
  "_data": {
    "sourceId": "arxiv",
    "paperId": "2310.18313",
    "url": "https://arxiv.org/abs/2310.18313",
    "title": "FP8-LM: Training FP8 Large Language Models",
    "authors": "Houwen Peng, Kan Wu, Yixuan Wei, Guoshuai Zhao, Yuxiang Yang, Ze Liu, Yifan Xiong, Ziyue Yang, Bolin Ni, Jingcheng Hu, Ruihang Li, Miaosen Zhang, Chen Li, Jia Ning, Ruizhe Wang, Zheng Zhang, Shuguang Liu, Joe Chau, Han Hu, Peng Cheng",
    "abstract": "In this paper, we explore FP8 low-bit data formats for efficient training of large language models (LLMs). Our key insight is that most variables, such as gradients and optimizer states, in LLM training can employ low-precision data formats without compromising model accuracy and requiring no changes to hyper-parameters. Specifically, we propose a new FP8 automatic mixed-precision framework for training LLMs. This framework offers three levels of FP8 utilization to streamline mixed-precision and distributed parallel training for LLMs. It gradually incorporates 8-bit gradients, optimizer states, and distributed learning in an incremental manner. Experiment results show that, during the training of GPT-175B model on H100 GPU platform, our FP8 mixed-precision training framework not only achieved a remarkable 39% reduction in real memory usage but also ran 75% faster than the widely adopted BF16 framework (i.e., Megatron-LM), surpassing the speed of Nvidia Transformer Engine by 37%. This largely reduces the training costs for large foundation models. Furthermore, our FP8 mixed-precision training methodology is generic. It can be seamlessly applied to other tasks such as LLM instruction tuning and reinforcement learning with human feedback, offering savings in fine-tuning expenses. Our FP8 low-precision training framework is open-sourced at {this https URL}{this http URL}.",
    "timestamp": "2025-08-08T18:18:03.690Z",
    "rating": "novote",
    "publishedDate": "2023/10/27",
    "tags": [
      "Machine Learning (cs.LG)",
      "Computation and Language (cs.CL)"
    ],
    "doi": "",
    "journalName": "",
    "sourceType": "url"
  },
  "_meta": {
    "client_version": "0.11.1",
    "timestamp": "2025-08-08T18:18:04.313Z",
    "update_mode": "append",
    "issue_number": 4
  },
  "type": "initial_state"
}
